src/pages/nft/claim/index.vue

```diff
@@ -490,8 +490,12 @@
           <ButtonV2
             :content-class="['px-[48px]']"
             class="phoneLg:w-full phoneLg:max-w-[480px]"
-            preset="tertiary"
-            :text="$t('nft_claim_claimed_button_view_collection')"
+            :preset="isViewCollectionLoading ? 'plain' : 'tertiary'"
+            :text="
+              isViewCollectionLoading
+                ? $t('settings_follow_loading')
```

Comment:
Can we not reuse another translation key for this?

Reply:
[🎨 Update translation key](https://github.com/likecoin/liker-land/pull/1773/commits/ab945b9d23ad54c29e105caabb85adc149dea51b)